StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/CVCReconfirmationViewController.swift

```diff
@@ -94,6 +94,14 @@ class CVCReconfirmationViewController: UIViewController {
         ])
         updateUI()
     }
+
+    #if !STP_BUILD_FOR_VISION
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        self.cvcFormElement.traitCollectionDidChange(previousTraitCollection)
```

Comment:
IMO instead of manually doing this, we should make a CVCRecollectionView that subclasses UIView so it can just override `traitCollectionDidChange` like other element views.

Reply:
Chatted offline -- thanks for the help! great suggestion!